corehq/apps/locations/resources/v0_1.py

```diff
@@ -56,7 +56,7 @@ def obj_get_list(self, bundle, **kwargs):
         domain = kwargs['domain']
         project = getattr(bundle.request, 'project', self.domain_obj(domain))
         parent_id = bundle.request.GET.get('parent_id', None)
-        include_inactive = json.loads(bundle.request.GET.get('include_inactive', 'false'))
+        include_inactive = json.loads(bundle.request.GET.get('include_inactive') or 'false')
```

Comment:
This seems a little...unconventional...because it is. 

I encountered an "issue" where `bundle.request.GET.get('include_inactive')` would return `''` and did not default to `'false'` - I'm not sure why. The easiest way to get around that was to then simply do the `or` logic.

Reply:
Would this do what you want? https://github.com/dimagi/commcare-hq/blob/d7a4a7f17ba8d44db48114687b39afb17160d889/corehq/ex-submodules/dimagi/utils/parsing.py#L10-L22


```suggestion
        try:
            include_inactive = string_to_boolean(bundle.request.GET.get('include_inactive', False))
        except ValueError:
            include_inactive = False
```

Maybe that's less readable than what you have here.

Reply:
I second the idea of using `string_to_boolean()` because it's more consistent in terms of the truthy/falsey values that we accept than using `json.loads()`.

We might be able to drop the `try` block, because I think `bundle.request.GET.get('include_inactive')` is always going to return a string or `None`. (Unless I'm missing something?)
```python
>>> from django.http import QueryDict
>>> qd = QueryDict('a=1&a=2&c=3')
>>> qd.get('a')
'2'
>>> type(qd.get('d'))
NoneType
```

Reply:
This seems more Python-esque.